components/logins/src/login.rs

```diff
@@ -336,7 +336,8 @@ impl LoginFields {
                 }
                 Ok(None)
             }
-            Err(_) => {
+            Err(e) => {
+                breadcrumb!("Error parsing login origin: {e:?}");
```

Comment:
I'm pretty sure that logging an [url::ParseError](https://docs.rs/url/latest/url/enum.ParseError.html) shouldn't leak any PII.

Reply:
Yeah IIUC this will only pass in the `ParseError` enum variant as a string so no other details will be passed. I'm going to approve with that assumption. If we find otherwise it'll be easy enough to revert this change.